components/x-follow-button/src/styles/mixins/lozenge/main.scss

```diff
@@ -1,6 +1,53 @@
 @import './themes';
 @import './toggle-icon';
 
+@mixin focusOutlineColor($focus-color) {
```

Comment:
This mixin is based on [o-nomalise _oNormaliseFocus mixin](https://github.com/Financial-Times/o-normalise/blob/master/src/scss/_mixins.scss#L107-L196)